### DISCUSSION TITLE
Consider adding a header image to the readme

### DISCUSSION THREAD

**Discussion by jcbhmr** [spenserblack/actions-wiki#29] [Ideas]
Something like this? Might want to make the filenames actually line up, but the quick "what does this action do" is nice. even if there's already a title, an image like this clearly gets across what this action does and would be a good addition IMO

![image](https://user-images.githubusercontent.com/61068799/225987228-827ccde8-0d5e-495f-aa1f-eefb87fc2e2c.png)

it's sort of like a blog post; they are better when they have an image, even if the image is only tangentially related to what the content is. I mean, just look at this: https://web.dev/blog/

![image](https://user-images.githubusercontent.com/61068799/225987799-03cc8e5b-ab53-4e35-a04a-acf99b2cf89a.png)

wtf are these images?! they have almost nothing to do with html or js, and yet it's _still better than just a plain text post_. it helps break it up, and adds some "oh yeah that one" sort of like a profile picture for the project. it's like a "meme" in the truest _technical_ sense of the definition.

**Comment by spenserblack** [comment]
Playing around with sizes, I think around `300px` is a good height. Doesn't force too much vertical scrolling.
Nitpick: since `.github/wiki` is the default directory, the tree should be `.github/wiki`.

Edit: or `320px` to be the bare minimum to include the image for the social preview :upside_down_face:

**Comment by jcbhmr** [comment]
closing discussion because the _discussion_ part is over, now we're onto the issue/todo part
